usecase/csv/csv.go

```diff
@@ -0,0 +1,49 @@
+package csv
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+	"bootcamp/domain/model"
+	"bootcamp/utils"
+	"github.com/gorilla/mux"
+)
+
+/*
+GetPokemon returns a Pokemon list read from a CSV file
+If query params exists or /{id}, filter the Pokemon
+Otherwise, returns the list
+*/
+func GetPokemon(r *http.Request) (model.PokemonList, error) {
+	pokemonList, err := utils.ReadCSV()
```

Comment:
Just like you have a service for DB, wouldn't it be a good idea to also have a service that handles reading and writing to the csv file?

The usecase should handle your business logic, like calling services that handle communication with DBs or external APIs, and then processing the information as required, and reading and writing to a csv is very similar to communicating with a DB, right?

All of that file manipulation stuff could go on a csv service, probably the functions you have on the `utils` package that handle that could go on a dedicated csv service

Reply:
*